torch/testing/_internal/common_quantization.py

```diff
@@ -294,6 +294,22 @@ def wrapper(*args, **kwargs):
             fn(*args, **kwargs)
     return wrapper
 
+def skipIfNoCaffe2AtenFallback(fn):
+    reason = 'Certain quantized operations require Caffe2 to be built-in.'
+    if isinstance(fn, type):
+        if not torch.onnx._CAFFE2_ATEN_FALLBACK:
+            fn.__unittest_skip__ = True
+            fn.__unittest_skip_why__ = reason
+        return fn
+
+    @functools.wraps(fn)
+    def wrapper(*args, **kwargs):
+        if not torch.onnx._CAFFE2_ATEN_FALLBACK:
+            raise unittest.SkipTest(reason)
+        else:
+            fn(*args, **kwargs)
+    return wrapper
+
```

Comment:
Hmm, this looks like almost an exact duplicate of the same method in `torch/testing/_internal/comon_utils.py`? Why not just import the same method? Or create `_skipIfNotCaffe2WithReason(fn, reason):` in common_utils, import it here and provide another reason?